src/pages/App.scss

```diff
@@ -74,6 +74,10 @@ a code {
   text-decoration: underline;
 }
 
+pre code {
```

Comment:
This ought to be vetted by someone more familiar with the project.

I wanted to use triple backticks to surround code blocks inside the markdown file for my example but the text was unreadable since bootstrap changes the color of `pre code` elements.